spec/schema.json

```diff
@@ -72,6 +71,39 @@
           }
         }
       }
+    },
+    "/confirmation-statement/company/{company_number}/eligibility": {
+      "parameters": [
+        {
+          "name": "company_number",
+          "in": "path",
+          "required" : true,
+          "type" : string
```

Comment:
there are a few other string like this one that need to have quotes around them, eg "string". if you look in intellij there are a handful of others.